CHANGELOG.md

```diff
@@ -25,6 +25,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.21.5] - 2023-05-08
+
+### Fixed
+- Updated container build script to pull in the correct kubectl version from the
+  metal-provision repository instead of the csm-rpms repository.
+
+### Security
+- Rebuilt the cray-sat container image to address CVE-2023-27536 in
```

Comment:
Did it also resolve the other CVEs Snyk was reporting? If so, we could add those here too. I also wonder about adding those to the integration/main branches.

Reply:
Snyk claims that there aren't any vulnerabilities in the built image: 

```
snyk container test artifactory.algol60.net/sat-docker/unstable/cray-sat:3.21.5-20230508215621_8bd6a20


Testing artifactory.algol60.net/sat-docker/unstable/cray-sat:3.21.5-20230508215621_8bd6a20...

Organization:      jack.stanekhpe.com
Package manager:   apk
Project name:      docker-image|artifactory.algol60.net/sat-docker/unstable/cray-sat
Docker image:      artifactory.algol60.net/sat-docker/unstable/cray-sat:3.21.5-20230508215621_8bd6a20
Platform:          linux/amd64
Base image:        alpine:3.15.8
Licenses:          enabled

✔ Tested 71 dependencies for known issues, no vulnerable paths found.

According to our scan, you are currently using the most secure version of the selected base image```